source/_components/sensor.sensehat.markdown

```diff
@@ -34,56 +35,13 @@ Configuration variables:
   - 'temperature'
   - 'humidity'
   - 'pressure'
+  is_hat_attached (Optional): True|False boolean; True declares the SenseHAT _is_ physically on the Raspberry Pi
+  
 
-### {% linkable_title Installation instruction for the All-In-One installer and HASSbian %}
+#### Customizing the Sense HAT data
```

Comment:
This part was already added by https://github.com/home-assistant/home-assistant.github.io/pull/2009. I suggest that you limit your PR to the new Boolean and make it against `next` which is the right branch for new features.